networking/gcp/psa/examples/simple/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "region" {
+  description = "The GCP region to deploy the Artifact registry"
+  type        = string
+  default     = "europe-west9"
+}
+
+variable "project" {
+  description = "Project name"
+  type        = string
+  default     = "armonik-gcp-13469"
```

Comment:
```suggestion
  default     = "armonik-gcp"
```

Reply:
```suggestion
  default     = null
```

If no project is specified, the currently configured project should be used (automatic).

Reply:
When I do this, I get an error (see below) when planning the examples
![image](https://github.com/aneoconsulting/ArmoniK.Infra/assets/136320932/6405977f-b4bb-46a2-a8fc-18f22db31d33)